lib/syskit/test/profile_assertions.rb

```diff
@@ -343,7 +349,7 @@ def assert_can_instanciate_together(*actions)
                 )
             rescue NetworkGeneration::PartialNetworkResolution => e
                 raise ProfileAssertionFailed.new("configure together", actions,
-                                                 e.original_exceptions)
+                                                 *e.original_exceptions)
```

Comment:
Adding the splat will cause a "wrong number of arguments" errors if there is more than one original_exceptions. `initialize` expects only 3 arguments.

Keep initialize as it is and remove the splat here.